.gitignore

```diff
@@ -135,19 +135,9 @@ ibis/examples/descriptions
 ci/odbc/odbc.ini
 *-citibike-tripdata.tar.xz
 
-# data downloaded by the geospatial tutorial
-docs/posts/ibis-duckdb-geospatial/nyc_data.db.wal
```

Comment:
Good idea, moving all this stuff into a `docs/.gitignore`. I definitely forget about simple solutions to `.gitignore` sprawl!